### ISSUE TITLE
[Bug]: Pivot overflow menu aria-required-children failure

### ISSUE THREAD

**Issue by gfisher-vmsft** [microsoft/fluentui#26951]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
All browsers
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/v-grfisher/pen/QWVEmpy

## Repro Steps
1. Navigate to [Pivot with Overflow Menu](https://developer.microsoft.com/en-us/fluentui#/controls/web/pivot) section 
2. Run fastpass

## Actual Behavior
- fastpass error: aria-required-children (role="tab" missing)

## Expected Behavior
- no fastpass errors

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by astherath** [comment]
Hey all, is there an ETA on this bugfix? We have a few blocking fastpass issues with this component. Thanks!!

**Comment by rogerprz** [comment]
any update? @smhigley

**Comment by smhigley** [comment]
This bug exists because the ARIA tablist pattern does not currently support having an overflow menu (it's a known problem, and being worked on). Because we have an overflow menu, we have two options to implement it:
1. The menu trigger is a menubutton, which is technically not an allowed child of a tablist (though negative user impact here is very minimal, since the menubutton triggers the correct mode in windows screen readers, similar to tabs).
2. The menu trigger is a tab with `aria-haspopup`, which is not an allowed attribute on that role. Also not much negative user impact here.

Neither choice is perfect; we chose the first for v8 and the second would also be fine, though both would give you automated errors. Until whatever year this pattern actually gets supported in the spec, those errors should be ignored.
